qdrant-landing/content/documentation/guides/capacity-planning.md

```diff
@@ -0,0 +1,103 @@
+- **Boolean fields** typically consume 1 byte. 
+
+<aside role="alert">
+    The easiest way to calculate your payload size is to use a JSON size calculator.
+</aside>
+
+Calculating total payload size is similar to vectors. We have to multiply it by 1.5 for back-end indexing processes.
+
+```text
+total_payload_size = number_of_points * payload_size * 1.5 
+```
+
+Let's say you want to store 1 million points with JSON payloads of 5KB:
+
+```text
+total_payload_size = 1,000,000 * 5KB * 1.5 
+```
+The total_payload_size is approximately 5,000,000 bytes, or about 4.77 GB.
+
+## Choosing disk over RAM
+
+For optimal performance, you should store only frequently accessed data in RAM. The rest should be offloaded to the disk. For example, extra payload fields that you don't use for filtering can be stored on disk. 
+
+Only [indexed fields](../../concepts/indexing/#payload-index) should be stored in RAM. You can read more about payload storage in the [Storage](../../concepts/storage/#payload-storage) section.
+
+### Storage-focused configuration
+
+If your priority is to handle large volumes of vectors with average search latency, it's recommended to configure [memory-mapped (mmap) storage](/documentation/concepts/storage/#configuring-memmap-storage). In this setup, vectors are stored on disk in memory-mapped files, while only the most frequently accessed vectors are cached in RAM.
+
+The amount of available RAM greatly impacts search performance. As a general rule, if you store half as many vectors in RAM, search latency will roughly double.
+
+Disk speed is also crucial. [Contact us](/documentation/support/) if you have specific requirements for high-volume searches.
+
+### Subgroup-oriented configuration
+
+If your use case involves splitting vectors into multiple collections or subgroups based on payload values (e.g., serving searches for multiple users, each with their own subset of vectors), memory-mapped storage is recommended.
+
+In this scenario, only the active subset of vectors will be cached in RAM, allowing for fast searches for the most recent and active users. You can estimate the required memory size as:
+
+```text
+memory_size = number_of_active_vectors * vector_dimension * 4 bytes * 1.5
+```
+
+## Scaling disk space in Qdrant Cloud
+
+Clusters supporting vector search require substantial disk space. If you're running low on disk space, you can use the UI at [cloud.qdrant.io](https://cloud.qdrant.io/) to **Scale Up** your cluster.
+
+<aside role="status">Note: If you increase disk space via the Qdrant UI, you cannot reduce it later.</aside>
+
+When running low on disk space, consider the following benefits of scaling up:
+
+- **Larger Datasets**: Supports larger datasets, which can improve the relevance and quality of search results.
+- **Improved Indexing**: Enables the use of advanced indexing strategies like HNSW.
+- **Caching**: Enhances speed by allowing frequently accessed data to be cached on disk.
```

Comment:
We don't cache frequently accessed data on disk.

I assume scale up also implies more RAM. If so, I'd write it to specify that:

```suggestion
- **Caching**: Enhances speed by having more RAM, allowing more frequently accessed data to be cached.
```